assets/samplesheet_1_row_sample.csv

```diff
@@ -0,0 +1,2 @@
+sample,image_directory
+TEST1,/home/pollen/HITS/nextflow/test_mcmicro-nf-core/input_sample
```

Comment:
path to local data

Reply:
(See extensive discussion in Slack regarding the background for the local paths and some bigger-picture questions about what a test samplesheet should look like when the input is actually a directory not a file)
We don't actually end up doing anything with that `image_directory` column in the tests because they specify a list of the actual file paths that would have been contained in that directory as an explicit input channel for the pipeline:
https://github.com/nf-core/mcmicro/blob/eb69c1ca66171512749bca954236af9b472c07b9/tests/main.nf.test#L141-L153

The fact that the `input_sample` param needs to be passed to these tests at all is a holdover from our initial input validation approach, which re-parses the samplesheet in the main pipeline rather than `PIPELINE_INITIALISATION`. That defect is being fixed in a subsequent PR. Would you allow these sheets with local paths to pass for now, with the understanding that we'll address the issue soon (certainly before merging to master)?

Reply:
I ended up replacing all the local paths with URLs after all -- I realized we do have a way to support this already.